ci/Jenkinsfile

```diff
@@ -82,7 +82,10 @@ pipeline {
   stages {
     stage('Prep') {
       steps { script {
+        sh 'env | grep nightly'
+        sh 'env | grep BUILD_'
         setNewBuildName()
+        sh 'echo $BUILD_DISPLAY_NAME'
```

Comment:
Why did we merge these debug commands into `master`?

Reply:
As I remember I wanted to see results from some configurations, which I could not trigger from PR branch. This debug was removed few days after.